src/Files.App/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using CommunityToolkit.WinUI;
+using Microsoft.UI.Dispatching;
+using System;
+using System.Threading.Tasks;
+
+namespace Files.App.Extensions
+{
+	// Window.DispatcherQueue seems to be null sometimes.
+	// We don't know why, but as a workaround, we invoke the function directly if DispatcherQueue is null.
+	public static class DispatcherQueueExtensions
+	{
+		public static Task EnqueueOrInvokeAsync(this DispatcherQueue dispatcher, Func<Task> function, DispatcherQueuePriority priority = DispatcherQueuePriority.Normal)
```

Comment:
```suggestion
		public static Task EnqueueOrInvokeAsync(this DispatcherQueue? dispatcher, Func<Task> function, DispatcherQueuePriority priority = DispatcherQueuePriority.Normal)
```

Add nullable sign, and in other places too